neo/Wallets/Wallet.cs

```diff
@@ -202,7 +202,7 @@ public virtual WalletAccount Import(string nep2, string passphrase)
             return account;
         }
 
-        public Transaction MakeTransaction(TransferOutput[] outputs, UInt160 from = null)
+        public Transaction MakeTransaction(Snapshot snapshot, TransferOutput[] outputs, UInt160 from = null)
```

Comment:
I think we don't need to pass the `snapshot` to this method. Because when we need to make a transaction, we always wish to make it according to the current snapshot.

Reply:
Yes, is only for my unit test, you prefer to mock this?

Reply:
Better to pass on method, more options for us to test. Let's decouple the code, please.

Reply:
No. Don't change the code for tests only.